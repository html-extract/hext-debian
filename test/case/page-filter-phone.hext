<div @text:filter(/Phone: ([0-9]+)/):phone_nr />
