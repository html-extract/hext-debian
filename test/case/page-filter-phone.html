<div>
  Phone: 012345689
</div>
<div>
  Phone: 0
</div>
<div>
</div>
