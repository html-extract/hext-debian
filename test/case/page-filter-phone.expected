{"phone_nr":"012345689"}
{"phone_nr":"0"}
{"phone_nr":""}
